docs/api/spec.md

```diff
@@ -78,29 +78,31 @@ Examples of End Points include:
 ### Open an Application by Name
 Linking from one application to another is a critical basic workflow that the web revolutionized via the hyperlink.  Supporting semantic addressing of applications across different technologies and platform domains greatly reduces friction in linking different applications into a single workflow.
 
-### Raising Intents
+### Raising Intents or Contexts
```

Comment:
This could be changed back to just `Raising Intents` - there is not an additional concept to understand.